### PR TITLE
Update to Warzone2100-3.3.0

### PR DESCRIPTION
**Pull request by cozybuild**

### CODE REVIEW

**Comment by cozybuild** [comment]
bot, build

**Comment by cozybuild** [comment]
bot, build

**Comment by corecontingency** [comment]
bot, build

File: net.wz2100.wz2100.yaml [hunk 1]
```diff
@@ -35,25 +35,25 @@ modules:
...
   - name: fribidi
     config-opts:
     - --disable-docs
     sources:
       - type: git
         url: https://github.com/fribidi/fribidi
-        tag: v1.0.5
-        commit: 5b6a16e8da12ae7ff482fbfa5a17b72bd518418f
+        tag: v1.0.7
+        commit: aac3845708969b9c4bc2874edc4950123595b1b4
```
Comment (corecontingency) [inline net.wz2100.wz2100.yaml:48]: Fribidi has updated to version v1.0.8 while waiting for someone to merge this pull request, unfortunately.

**Review by corecontingency — APPROVED** [review]
Just tested it, and seems to work great. Using Arch Linux, with GNOME running via Wayland. No problems with fullscreen, loaded up the campaign, a skirmish match, and the introduction video and everything runs fine, no problems. Any reason why this isn't being merged?

Also, [fribidi](https://github.com/fribidi/fribidi/releases) updated to version 1.0.8 while waiting for this request to be merged, so maybe that should be updated first.

**Comment by bilelmoussaoui** [comment]
bot, build

File: net.wz2100.wz2100.yaml (hunk 1 above)
Comment (bilelmoussaoui) [reply net.wz2100.wz2100.yaml:48]: Will merge this. Mind sending a merge request to update Fribidi? Thank you

**Pull request merged**